packages/pieces/framework/src/lib/apps/google-calendar/actions/create-quick-event.ts

```diff
@@ -18,7 +18,7 @@ export const createQuickCalendarEvent = createAction({
       description: 'The text describing the event to be created',
       required: true,
     }),
-    send_updates: Property.Dropdown<string>({
+    send_updates: Property.Dropdown<string, false>({
```

Comment:
Optional Dropdown will require to specficy the second argument now, @khaledmashaly not sure If we should consider other alternative like

Property.Optional({})

What do you think?

Reply:
I just found this could be avoided by making default value is boolean instead of true :) :) :)